chef/cookbooks/network/recipes/default.rb

```diff
@@ -209,6 +209,16 @@ def get_datapath_id_for_ovsbridge(bridge)
     ifs[bond.name]["addresses"] ||= Array.new
     ifs[bond.name]["slaves"] = Array.new
     base_ifs.each do |i|
+      # If the interface is configured, wicked will helpfully re-add the IP
+      # when the interface is downed and then brought back up -- which is
+      # required, since interfaces which are up can not be added as slaves.
+      # To stop this happening, remove the configuration file and reload the
+      # interface.
+      if ::File.exist?("/etc/sysconfig/network/ifcfg-#{i.name}")
```

Comment:
Unfortunately we can't do it this way. As it means that we will remove the config file and do a wicked ifreload with every single chef-client run. (As the config file for the slave is being recreated further down in the recipe).

Unfortunately I don't have a good idea for a solution to the problem myself yet. It's a bit of a mess. I guess we need to have a discussion with the wicked guys.